src/js/node/http.ts

```diff
@@ -1295,7 +1300,13 @@ class ClientRequest extends OutgoingMessage {
         verbose: !!$debug,
         signal: this[kAbortController].signal,
         proxy: proxy,
-
+        ssl_props: {
```

Comment:
we should check if we are calling https before accepting/passing these options to fetch, we can check to give the same error as nodejs

Reply:
Whats the error we should throw

Reply:
someting like `ca param is only valid for https requests`